packages/typescript/install.md

```diff
@@ -141,9 +141,9 @@ nodejs_binary(
 then refer to that target in the `compiler` attribute of your `ts_library` rule.
 
 Note that `nodejs_binary` targets generated by `npm_install`/`yarn_install` can include data dependencies
-on packages which aren't declared as dependencies. For example, if you use [tsickle] to generate Closure Compiler-compatible JS, then it needs to be a `data` dependency of `tsc_wrapped` so that it can be loaded at runtime.
-￼
-[tsickle]: https://github.com/angular/tsickle
+on packages which aren't declared as dependencies. For example, if you use [tsickle](https://www.npmjs.com/package/tsickle) to generate Closure Compiler-compatible JS, then it needs to be a data dependency of `tsc_wrapped` so that it can be loaded at runtime.
+
+[tsickle](https://github.com/angular/tsickle)
```

Comment:
this one was correct before - actually there was just a garbage character non line 145 messing up the formatting